doc/developers/demo.rst

```diff
@@ -18,27 +19,26 @@ This script automates the steps described in :doc:`/operators/start_network`, in
 - verifying that each node has successfully joined the new service
 - proposing and passing governance votes using the generated member identities
 
-The following command will run a simple 3-node test network on a single machine:
+The following command will run a simple one node test network on a single machine:
 
 .. code-block:: bash
 
     $ cd CCF/build
 
-    $ ../start_test_network.sh --package ./liblogging.enclave.so.signed
+    $ ../sandbox.sh -p ./liblogging.virtual.so
```

Comment:
```suggestion
    $ ../tests/sandbox.sh -p ./liblogging.virtual.so
```

`start_test_network.sh` was under the project root, and I think we also copied it into the build directory. Perhaps we should be doing that with `sandbox.sh` as well, but I think that can be implicit in the "make the build dir look like the install dir" task.